Update install profile.

leftclickben opened the pull request:
- Add various list items created by import.
- Add Loan Status field and list.

leftclickben committed:
Update install profile.

+ Add various list items created by import.
+ Add Loan Status field and list.

kehh commented:
Could you remove the list items with semicolons in them and change the locale to en_AU.

leftclickben committed:
Fix issues with generated profile.

+ Change locale en_US => en_AU.
+ Remove invalid list items.
+ Remove empty restriction entries.

leftclickben commented:
Done and build appears to be passing.
